src/features/attribute/use_cases/get_attribute_use_case.py

```diff
@@ -18,5 +18,9 @@ def get_attribute_use_case(
     direct_address = address
     if resolve and node.attribute.attribute_type == SIMOS.REFERENCE.value:
         direct_address = node.entity["address"]
-        node: Node = document_service.get_document(Address.from_absolute(direct_address), 2)  # type: ignore
+        # type: ignore
+        reference_node: Node = document_service.get_document(
+            Address.from_relative(direct_address, None, Address.from_absolute(address).data_source), 2
```

Comment:
Could we add a few comments to this?
I'm really struggling to see what is going on :smiling_face_with_tear:

Reply:
Like what are the different addresses, why are we calling "from_relative()" on `direct_address` (that _sounds_ wrong, even if it probably isn't

Reply:
Comments would probably be nice, yes.
I'm honestly not quite sure whats going on here myself, so maybe @eoaksnes can chime in and explain a bit better, but in essence;

When referencing an entity that references a file, you somehow lose the datasource somewhere... No idea why...

We use the `from_relative`, because that one uses the `from_absolute` if the `direct_address` contains a data source..

Also, doing this fixed the issue babak was having with images (I hope), and we were struggling with this for quite some time 😅

Reply:
No, I think the issue is that its not a part of the file id then... 🤔 We can look at this together tomorrow 👍

Reply:
@eoaksnes tagging you here since you have more insight into this than I do :+1: